docs/versionhistory.rst

```diff
@@ -1,6 +1,12 @@
 Version history
 ===============
 
+**UNRELEASED**
+
+- Modified socket tests to generate unix socket paths of lengths less than 107 bytes.
+  Modern unices enforce this limitation and throw an OSError when the path is too long
+  (PR by Colin Taylor)
+
```

Comment:
There's no need for a changelog entry as this is just an internal change.
```suggestion
```

Reply:
Just being thorough - removed.